### DISCUSSION TITLE
Map of ReEDS regions

### DISCUSSION THREAD

**Discussion by keurek** [NREL/ReEDS-2.0#18] [Q&A: Data] [answered]
Where can I find a map of the ReEDS regions?

**Comment by keurek** [comment, marked as answer]
A map of the 356 resource regions and 134 balancing areas for the conterminous United States can be found on NREL's website: https://www.nrel.gov/analysis/assets/pdfs/reeds-model-regions-map.pdf

**Reply by Annafache** [reply]
Where can I find more detailed information about the breakdown of the different balancing areas?

**Reply by wesleyjcole** [reply]
There is a hierarchy.csv file at bokehpivot\in\reeds2\hierarchy.csv.  It includes a mapping to other regions that might be of interest.

All of the balancing areas are built from counties, so if you need a mapping of counties to balancing areas let us know.  We can also provide shape files of the regions if needed.

**Reply by keurek** [reply]
The shape files for the model balancing areas are now available in the repository:
https://github.com/NREL/ReEDS_OpenAccess/tree/master/spatial_data

**Reply by Annafache** [reply]
Hi, 
Thanks for all of this. I'm trying to figure out how the county of Florida is separated in the p101 and p102. How would I determine that?

**Reply by vcelloho** [reply]
The shp files that are part of the spatial_data folder can be loaded into GIS software like ArcGIS or the free QGIS. If you also load a shp file of counties available from the US Census you can determine the county separation either by inspection of the overlayed spatial data or by preforming an intersection between the two layer features.
https://www.census.gov/geographies/mapping-files/time-series/geo/carto-boundary-file.html
https://docs.qgis.org/2.18/en/docs/user_manual/processing_algs/qgis/vector_overlay_tools.html#intersection

**Reply by keurek** [reply]
The attached spreadsheet is a crosswalk of counties with the 356 ReEDS resource regions and 134 balancing areas.
[county_reeds_20120310.xlsx](https://github.com/NREL/ReEDS_OpenAccess/files/6632531/county_reeds_20120310.xlsx)

**Reply by yojowe-hash** [reply]
Do the ReEDS regions correspond to utility service territories/load ones and/or hubs? If so, is there an index available?

**Reply by yojowe-hash** [reply]
Found my answer https://github.com/NatLabRockies/ReEDS-2.0/discussions/43

**Comment by Annafache** [comment]
Thank you!
Anna
